uuid/uuid.cabal

```diff
@@ -30,20 +30,22 @@ Source-Repository head
     Subdir:            uuid
 
 Library
-    Build-Depends:     base            >= 4.3      && < 5
-                     , binary          >= 0.4      && < 0.9
-                     , bytestring      >= 0.10     && < 0.11
-                     , cryptohash-sha1 >= 0.11.100 && < 0.12
-                     , cryptohash-md5  >= 0.11.100 && < 0.12
-                     , entropy         >= 0.3.7    && < 0.5
-                     , network-info    == 0.2.*
-                     , random          >= 1.0.1    && < 1.2
-                     , time            >= 1.1      && < 1.9
-                     , text            >= 1.2.3    && < 1.3
-                     , uuid-types      >= 1.0.2    && < 2
+    Build-Depends:     base             >= 4.3      && < 5
+                     , binary           >= 0.4      && < 0.9
+                     , bytestring       >= 0.10     && < 0.11
+                     , cryptohash-sha1  >= 0.11.100 && < 0.12
+                     , cryptohash-md5   >= 0.11.100 && < 0.12
+                     , entropy          >= 0.3.7    && < 0.5
+                     , network-info     == 0.2.*
+                     , random           >= 1.0.1    && < 1.2
+                     , template-haskell >= 2.7      && < 3
```

Comment:
The upper bound `< 3` albeit looking cute isn't compliant w/ https://pvp.haskell.org/